{"uid":"d31f2c93a7a41693","name":"test_address_prefix[Test for Snow]","fullName":"tests.test_network_parameters.TestNetworkPrefix#test_address_prefix","historyId":"3ae8cf1c4024df7b1e713b1f22304fc4","time":{"start":1669725627483,"stop":1669725627876,"duration":393},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7f982b5d1d30>\nchain = <tests.utils.chain_model.Chain object at 0x7f982b621790>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f982b621790>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Snow\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7f982b5d1d30>\nchain = <tests.utils.chain_model.Chain object at 0x7f982b621790>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f982b621790>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Snow\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"f79d4ee78bdf73ed","name":"stdout","source":"f79d4ee78bdf73ed.txt","type":"text/plain","size":54}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestNetworkPrefix"},{"name":"host","value":"fv-az304-876"},{"name":"thread","value":"1624-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7f982b621790>"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"53aaeff19d22d49d","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669725617047,"stop":1669725617474,"duration":427}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":24,"skipped":0,"passed":187,"unknown":0,"total":211},"items":[{"uid":"dcb7beb4b349008","reportUrl":"https://nova-wallet.github.io/test-runner/1988//#testresult/dcb7beb4b349008","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669710401445,"stop":1669710401838,"duration":393}},{"uid":"5d04306c669e677b","reportUrl":"https://nova-wallet.github.io/test-runner/1986//#testresult/5d04306c669e677b","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669684822699,"stop":1669684823213,"duration":514}},{"uid":"98ad4fc98f20dfb3","reportUrl":"https://nova-wallet.github.io/test-runner/1985//#testresult/98ad4fc98f20dfb3","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669666977248,"stop":1669666977657,"duration":409}},{"uid":"d48e67d43b7e3760","reportUrl":"https://nova-wallet.github.io/test-runner/1984//#testresult/d48e67d43b7e3760","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669652669080,"stop":1669652669484,"duration":404}},{"uid":"ae117975e83468f5","reportUrl":"https://nova-wallet.github.io/test-runner/1983//#testresult/ae117975e83468f5","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669638980286,"stop":1669638980992,"duration":706}},{"uid":"ec6c4de38e88806e","reportUrl":"https://nova-wallet.github.io/test-runner/1982//#testresult/ec6c4de38e88806e","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669623986668,"stop":1669623987052,"duration":384}},{"uid":"f8d00448a99b4c26","reportUrl":"https://nova-wallet.github.io/test-runner/1981//#testresult/f8d00448a99b4c26","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669609530524,"stop":1669609530912,"duration":388}},{"uid":"ea5a08239e7a79d8","reportUrl":"https://nova-wallet.github.io/test-runner/1980//#testresult/ea5a08239e7a79d8","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669598240840,"stop":1669598241205,"duration":365}},{"uid":"8f8bade0c0c7e4bd","reportUrl":"https://nova-wallet.github.io/test-runner/1979//#testresult/8f8bade0c0c7e4bd","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669580518136,"stop":1669580518679,"duration":543}},{"uid":"a924416eb5b1fea9","reportUrl":"https://nova-wallet.github.io/test-runner/1978//#testresult/a924416eb5b1fea9","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669566456630,"stop":1669566457328,"duration":698}},{"uid":"34c93d7018b5a063","reportUrl":"https://nova-wallet.github.io/test-runner/1977//#testresult/34c93d7018b5a063","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669552367207,"stop":1669552367584,"duration":377}},{"uid":"debc3ac443923e0f","reportUrl":"https://nova-wallet.github.io/test-runner/1976//#testresult/debc3ac443923e0f","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669537498493,"stop":1669537498994,"duration":501}},{"uid":"88bd4639734e42a4","reportUrl":"https://nova-wallet.github.io/test-runner/1975//#testresult/88bd4639734e42a4","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669523089726,"stop":1669523090246,"duration":520}},{"uid":"ddc4a7f78d49820a","reportUrl":"https://nova-wallet.github.io/test-runner/1973//#testresult/ddc4a7f78d49820a","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669493509522,"stop":1669493510050,"duration":528}},{"uid":"487f7494807358b5","reportUrl":"https://nova-wallet.github.io/test-runner/1972//#testresult/487f7494807358b5","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669479127784,"stop":1669479128310,"duration":526}},{"uid":"7789f3f82fb322e5","reportUrl":"https://nova-wallet.github.io/test-runner/1971//#testresult/7789f3f82fb322e5","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669464746533,"stop":1669464747021,"duration":488}},{"uid":"9ca4f7deba3c5cda","reportUrl":"https://nova-wallet.github.io/test-runner/1970//#testresult/9ca4f7deba3c5cda","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669450329618,"stop":1669450330119,"duration":501}},{"uid":"88409271b400b546","reportUrl":"https://nova-wallet.github.io/test-runner/1969//#testresult/88409271b400b546","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669435932360,"stop":1669435933049,"duration":689}},{"uid":"3d19b120d50c93f8","reportUrl":"https://nova-wallet.github.io/test-runner/1968//#testresult/3d19b120d50c93f8","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669422036476,"stop":1669422036842,"duration":366}},{"uid":"ab796fc1d28419d7","reportUrl":"https://nova-wallet.github.io/test-runner/1967//#testresult/ab796fc1d28419d7","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Snow","time":{"start":1669407103333,"stop":1669407103826,"duration":493}}]},"tags":[]},"source":"d31f2c93a7a41693.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7f982b621790>"]}